apps/grow_bitcoin/sources/grow_bitcoin.move

```diff
@@ -365,13 +395,48 @@ module grow_bitcoin::grow_bitcoin {
         table::add(&mut farming_asset.stake_table, asset_id, account);
     }
 
+    /// Supports batch invocation by stake users, allowing multiple assets to be unstaked to receive yield farming tokens
+    public entry fun batch_unstake(
+        signer: &signer,
+        assets: vector<ObjectID>,
+    ) {
+        let len = vector::length(&assets);
+        let i = 0;
+        let total_coin = coin::zero<GROW>();
+        while (i < len) {
+            let asset_id = *vector::borrow(&assets, i);
+            let asset = object::borrow_mut_object<UTXO>(signer, asset_id);
+            let coin = do_unstake(signer, object::id(asset));
+            utxo::remove_temp_state<StakeInfo>(asset);
+            coin::merge(&mut total_coin, coin);
+            i = i + 1;
+        };
+        account_coin_store::deposit(sender(), total_coin);
```

Comment:
Get the address from the signer via `signer::address_of(signer)` because if we call the `batch_unstake` via a contract account, the sender is different from the signer. Please also fix the `unstake` and `unstake_bbn` function.